src/components/AvailableImagesTile.js

```diff
@@ -12,7 +12,7 @@ import PropTypes from 'prop-types';
 import { useSelector, shallowEqual } from 'react-redux';
 
 const AvailableImageTileBase = ({ children }) => (
-  <Card>
+  <Card style={{ height: '100%' }}>
```

Comment:
I don't really like inline styling, would be much better to move this to a SCSS file (either a new one or use `App.scss`)